### PR TITLE
Add translated micro learning blog

### PR DESCRIPTION
**Pull request by sdegeus**
Translated blog

### CODE REVIEW

File: source/blog/nl/2017-08-07-micro-toekomst.html.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+Notendop-kennis. Ooit van gehoord? Heel kort gezegd is het het microdoseren van dagelijkse kennis. Hoewel **microleren** geen nieuw begrip is, vinden steeds meer mensen (waaronder ikzelf) het erg belangrijk
+
+## Informatiestroom
+
+Zie je wat er gebeurt?
+![Informatie tijd kennis](/images/blog/en/clocks.gif)
+
+We hebben tegenwoordig een razendsnelle informatieflow en maar 24 uur per dag. Door het internet is het snel delen van informatie mogelijk geworden. Dit zorgt voor een overvloed aan kennis en daardoor hebben we een paar stappen terug moeten zetten. Het was allemaal niet meer bij te benen.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-08-07-micro-toekomst.html.markdown]: en nog steeds maar 24 uur per dag

File: source/blog/nl/2017-08-07-micro-toekomst.html.markdown [hunk 2]
```diff
@@ -3,7 +3,8 @@ title: "Leren op microniveau: de toekomst"
 date: 2017-08-07 08:00 CEST
 tags: micro leren, elearning, kennis
 author: Milda
-lang: en
+lang: nl
+featured: true
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-08-07-micro-toekomst.html.markdown]: And remove from the other older blog

File: source/blog/nl/2017-08-07-micro-toekomst.html.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Leren op microniveau: de toekomst"
+date: 2017-08-07 08:00 CEST
+tags: micro leren, elearning, kennis
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-08-07-micro-toekomst.html.markdown]: Better tags are: 

microleren, e-learning, kennis

E-learning is used before. 

For overview of all tags that are used in the past, see http://www.defacto.nl/blog/tags/

**Comment by sdegeus** [comment]
Changes made, featured added and removed from the other blog, language changed and text edit made

**Comment by RonaldDijkstra** [comment]
Ok great, just the tags left.

**Comment by sdegeus** [comment]
Tag changes made

**Pull request merged**